assignments/callbacks.js

```diff
@@ -27,29 +27,65 @@ const items = ['Pencil', 'Notebook', 'yo-yo', 'Gum'];
 
 function getLength(arr, cb) {
   // getLength passes the length of the array into the callback.
+  return cb(arr);
```

Comment:
Your callback needs to receive the length of `arr`. Return `cb(arr.length)` inside your getLength function.